src/control/zoom/ZoomControl.cpp

```diff
@@ -110,22 +112,23 @@ void ZoomControl::zoomScroll(ZoomDirection zoomIn, utl::Point<double> zoomCenter
         startZoomSequence(zoomCenter);
     }
 
-    double newZoom = this->zoom + (zoomIn ? this->zoomStepScroll : -this->zoomStepScroll);
+    double newZoomStep = zoomIn ? this->zoomStepScroll : -this->zoomStepScroll;
+    double newZoom = this->zoom * std::exp(0.5 * newZoomStep);
     this->zoomSequenceChange(newZoom, false);
 }
 
 void ZoomControl::startZoomSequence() {
     Rectangle rect = getVisibleRect();
-    startZoomSequence({rect.x + rect.width / 2.0, rect.y + rect.height / 2.0});
+    startZoomSequence({rect.width / 2.0, rect.height / 2.0});
```

Comment:
Why don't we need `rect.x` and `rect.y` here?

Reply:
I changed `startZoomSequence` to use window coordinates not page relative coordinates. AFAIK `rect.x` would make it page relative. I tested it and it works this way.

Reply:
Okay! Thank you!